src/commands/game_commands/play.ts

```diff
@@ -1153,6 +1181,28 @@ export default class PlayCommand implements BaseCommand {
             }
         }
 
+        if (
+            gameType === GameType.CLIP &&
+            (!guildPreference.gameOptions.guessTimeout ||
+                guildPreference.gameOptions.guessTimeout >
+                    GuessTimeoutCommand.TIMER_MAX_ACCEPTABLE_CLIP_BEFORE_RESET)
+        ) {
+            await guildPreference.setGuessTimeout(
```

Comment:
What happens if they try to set the guess timeout after the game starts already?

Reply:
It's user configurable during the game and only overwritten when it's either not set, or set to a value too large